spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -2,12 +2,6 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
```

Comment:
Why this change?

Reply:
It confusingly brings in a completely different version that may or may not match what's in the 2021 BOM. I can back the change out, but then we should update the version. `spring-cloud-build` version in 2021.0.1 BOM is 3.1.1, and we here are still on 3.0.4.

It's also irrelevant because [`spring-cloud-dependencies-parent`](https://github.com/spring-cloud/spring-cloud-build/blob/v3.0.4/spring-cloud-dependencies-parent/pom.xml) does not contain anything except for license/scm/distribution and the snapsot repositories.

Reply:
Maybe let's just update the version for now, and do the removal in a different PR if needed?